app/Models/ResourceItem.php

```
@@ -32,39 +28,72 @@ class ResourceItem extends Model
+            ->where('id', '!=', $this->id)
+            ->pluck('task_id');
+        self::whereHas('task.phase', function ($query) use ($projectId) {
+            $query->where('project_id', $projectId);
+        })
+            ->where('resource_type', $this->resource_type)
+            ->where('unit', $this->unit)
+            ->where('description', $this->description)
+            ->where('id', '!=', $this->id)
+            ->update([
+                'unit_cost'  => $this->unit_cost,
+                'total_cost' => DB::raw('quantity * ' . (float) $this->unit_cost),
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'syncUnitCostAcrossProjectResources'. (Clean Code Rules)

(StaticAccess)